hyprctl/hyprctl.usage

```diff
@@ -10,13 +10,13 @@ hyprctl [<OPTIONS>]... <ARGUMENTS>
             |   (--batch)                                             "Execute a batch of commands separated by ;"
             ;
 
-<WINDOWS> ::= {{{ hyprctl clients | grep class | awk '{print $2}' }}}
+<WINDOWS> ::= {{{ hyprctl clients | grep class | awk '{print $2}' }}};
 
-<AVAILABLE_PLUGINS> ::= {{{ hyprpm list | grep "Plugin" | awk '{print $4}' }}}
+<AVIABLE_PLUGINS> ::= {{{ hyprpm list | grep "Plugin" | awk '{print $4}' }}};
```

Comment:
The previous spelling was correct.

Reply:
Right i war just that it wasn't corrected in the subcommand plugins it was still "AVIABLE_PLUGINS" and it didn't apply the completion now it should work

Reply:
I didn't re-uload the compiled completion files since it was compiled with both been name "AVIABLE_PLUGINS" and have tested and it worked